test/system/012-manifest.bats

```diff
@@ -143,6 +143,29 @@ EOF
     validate_instance_compression "2" "$list" "amd64" "zstd"
     validate_instance_compression "3" "$list" "arm64" "zstd"
 
+    smanifest="$list"
+    run_podman manifest inspect --authfile=$authfile --tls-verify=false \
+        $manifestpushed
+
+    objects=$(for obj in 0 1 2 3; do echo \
+          ".manifests[$obj].annotations" \
+          ".manifests[$obj].digest" \
+          ".manifests[$obj].platform.architecture" \
+          ".manifests[$obj].platform.os" \
+          ".manifests[$obj].mediaType" \
+          ".manifests[$obj].size" \
+          ; done)
+    for object in \
+        $objects \
+        '.schemaVersion' \
+        '.mediaType' \
+         ; do
```

Comment:
This is a really lovely clean way to express what could've been really ugly duplication. Thank you.